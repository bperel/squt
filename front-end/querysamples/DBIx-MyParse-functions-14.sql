SELECT CONVERT('string' USING latin1)
