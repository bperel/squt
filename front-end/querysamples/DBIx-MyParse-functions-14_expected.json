{
  "Warning": {
    "No alias": {
      "convert": ["function", "SELECT"]
    }
  },
  "Functions": {
    "0": {
      "to": "OUTPUT",
      "Constants": {
        "string": "string"
      },
      "name": "convert",
      "alias": "0"
    }
  }
}
